sysdig/data_source_sysdig_secure_composite_policy.go

```diff
@@ -0,0 +1,219 @@
+package sysdig
+
+import (
+	"context"
+	"strconv"
+	"strings"
+	"time"
+
+	v2 "github.com/draios/terraform-provider-sysdig/sysdig/internal/client/v2"
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceSysdigSecureCompositePolicy() *schema.Resource {
+	timeout := 5 * time.Minute
+
+	return &schema.Resource{
+		ReadContext: dataSourceSysdigSecureCompositePolicyRead,
+
+		Timeouts: &schema.ResourceTimeout{
+			Read: schema.DefaultTimeout(timeout),
+		},
+
+		Schema: createCompositePolicyDataSourceSchema(),
+	}
+}
+
+func dataSourceSysdigSecureCompositePolicyRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	return commonCompositePolicyDataSourceSecurePolicyRead(ctx, d, meta, "custom policy", isCustomCompositePolicy)
+}
+
+func isCustomCompositePolicy(policy v2.PolicyRulesComposite) bool {
+	return !policy.Policy.IsDefault && policy.Policy.TemplateId == 0
+}
+
+func createCompositePolicyDataSourceSchema() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		// IMPORTANT: Type is implicit: It's automatically added upon conversion to JSON
+		"type": {
+			Type:     schema.TypeString,
+			Computed: true,
+		},
+		"name":                  NameSchema(),
+		"description":           DescriptionComputedSchema(),
+		"enabled":               EnabledComputedSchema(),
+		"severity":              SeverityComputedSchema(),
+		"scope":                 ScopeComputedSchema(),
+		"version":               VersionSchema(),
+		"notification_channels": NotificationChannelsComputedSchema(),
+		"runbook":               RunbookComputedSchema(),
+		"rules": {
+			Type:     schema.TypeList,
+			Computed: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"id":          ReadOnlyIntSchema(),
+					"name":        ReadOnlyStringSchema(),
+					"enabled":     EnabledComputedSchema(),
+					"description": DescriptionComputedSchema(),
+					"tags":        TagsComputedSchema(),
```

Comment:
we don't have a concept of description/tags on malware rules (and i think all other composite policy types), did you find any that needed these fields?

Reply:
actually looks like we have tags but by default we set it to just "malware"

Reply:
and looks like rule description is just the policy name + `malware rule` when made through the UI. @ombellare do you think it's okay to let customers customize these 2 fields?

Reply:
Thanks @kmvachhani. We caught up with @ivanlysiuk-sysdig to clarify the changes. Tags have been hidden.